admin-cli/src/main/java/bio/terra/catalog/cli/Main.java

```diff
@@ -0,0 +1,119 @@
+package bio.terra.catalog.cli;
+
+import bio.terra.catalog.service.JsonValidationService;
+import bio.terra.catalog.service.dataset.DatasetDao;
+import bio.terra.catalog.service.dataset.DatasetId;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.networknt.schema.ValidationMessage;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Main {
+
+  private final DatasetDao datasetDao;
+  private final JsonValidationService jsonValidationService;
+  private final ObjectMapper objectMapper;
+
+  private final Map<String, Consumer<ApplicationArguments>> commands;
+
+  Main(
```

Comment:
I didn't create unit tests for this code yet. I'm not sure how much we need unit tests for an admin-only cli, although I can imagine we'd want unit tests for any admin operation that modifies the database.